src/sensor/ping360.cpp

```diff
@@ -122,6 +132,18 @@ void Ping360::handleMessage(const ping_message& msg)
 
     switch (msg.message_id()) {
 
+    case CommonId::DEVICE_INFORMATION: {
+        if(_configuring) {
+            startPreConfigurationProcess();
```

Comment:
Could there be any issues of this being called both here and by the QTimer?
Maybe restarting the timer beforehand is a good idea.

Reply:
Done.